etc/sonar.sh

```
@@ -17,15 +17,24 @@ if [ $(curl -I -s https://sonar-lightblue.rhcloud.com/ -s | head -n1 | grep "HTT
 fi
 
 
-# kill any previous port forward
+# kill any previous port forward and the previous mark file  about its status
 pkill -9 -f "rhc port-forward sonar [-]n lightblue"
+rm -f rhc-port-forward-status
 
-# forward ports from openshift (assumes current server is openshift online and you're a member of the lightblue namespace)
-rhc port-forward sonar -n lightblue &
+# forward ports from openshift (assumes current server is openshift online and you're a member of the lightblue namespace). A new mark file will eb created if the return value of the rhc command is different than 0
+rhc port-forward sonar -n lightblue || echo $? > rhc-port-forward-status  &
 
 # wait a bit for port forwarding to fire up
 sleep 10
 
+# if the file exists, log the error and exit 1
+if [ -f "rhc-port-forward-status" ]
```

This check can come before the `sleep 10` above so the script fails faster.